test/backtrace.jl

```diff
@@ -5,10 +5,29 @@ have_backtrace = false
 for l in bt
     lkup = ccall(:jl_lookup_code_address, Any, (Ptr{Void},), l)
     if lkup[1] == :backtrace
-        @test lkup[4] == false # fromC
+        @test lkup[6] == false # fromC
         have_backtrace = true
         break
     end
 end
 
-@test have_backtrace
```

Comment:
what happened to this test? might be good to add a test for the other properties in the backtrace to show that it found the right point in the file and didn't think it had inlining present.